README.md

```diff
@@ -505,7 +513,12 @@ Futher detail [here (staff only)](https://docs.google.com/document/d/1muN4pFqRhL
         <li>Leads collaborative discussions around evolving best practices, defining our engineering standards.</li>
         <li>Seeks out systemic problems and opportunities, and presents proposals to the Head of Engineering on how to, respectively, remediate them and benefit from them.</li>
         <li>Advises on new technology trials based on your previous experience, and mentor more junior developers in their technical discovery and technology intelligence.</li>
+        <li>Works with Product Managers and others to effectively communicate data-informed insights which inspire action from colleagues and stakeholders.†</li>
       </ul>
     </td>
   </tr>
 </table>
+
+† - Criteria marked with the 'obelisk' icon are new changes (2023-June) that are optional until after the current review round.
+The Engineering Management team are reviewing the progression and promotion frameworks generally, including adding data-related responsibilities. However we believe it's unfair to make changes to the criteria shortly before a review round.
```

Comment:
**Opinion**: this would read a little nicer in the positive. So something like...

```suggestion
The Engineering Management team are reviewing the progression and promotion frameworks generally, including adding data-related responsibilities. However we believe it's fair that changes to criteria are made and established well before a review round.
```